R/QC4b.R

```diff
@@ -34,7 +34,8 @@ QC4b <- function(d_metingen, verbose = F) {
   # selecteer pH veld en lab gegevens
   # afhankelijk van de dataset kan dit 'pH' of 'zuurgraad' zijn
   d <- d %>%
-    dplyr::filter(stringr::str_detect(parameter, "ph|zuurgraad")) #%>%
+    dplyr::filter(stringr::str_detect(parameter, 
```

Comment:
Deze str_detect is niet expliciet genoeg. Alles variabelen waairn 'pH' voor komt worden nu geselecteerd *dus ook pH_1_veld, ph_2_veld,  pH_veld en pH_onzin). Ik zou hier een expliciete dplyr::filter voor in de plaats zetten, eentje die checkt of de parameter 'pH' is of gelijk aan pH veldnaam.